src/main/java/frc/robot/commands/ArmPositionCommand.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.ArmSubsystem;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import java.util.function.Supplier;
+
+public class ArmPositionCommand extends CommandBase {
+  private final ArmSubsystem armSubsystem;
+  private final Supplier<Double> baseArmJoystick;
+  private final Supplier<Double> elbowArmJoystick;
+
+  public ArmPositionCommand(ArmSubsystem armSubsystem, Supplier<Double> baseArmJoystick, Supplier<Double> elbowArmJoystick) {
+    this.armSubsystem = armSubsystem;
+    this.baseArmJoystick = baseArmJoystick;
+    this.elbowArmJoystick = elbowArmJoystick;
+    addRequirements(armSubsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    //armSubsystem.updateMotorPositions(baseArmJoystick.get(), elbowArmJoystick.get());
```

Comment:
Hmm.  Seems like this command is no longer used or needed?  Should we remove the command and clean up robotcontainer as well?